client/components/App.vue

```diff
@@ -39,13 +41,29 @@ export default {
 		ConfirmDialog,
 		Mentions,
 	},
+	setup() {
```

Comment:
Can we and do we want to use https://vuejs.org/api/sfc-script-setup.html#script-setup?